wpilibcExamples/src/main/cpp/examples/HAL/c/Robot.c

```diff
@@ -63,7 +63,7 @@ int main(void) {
   // Create a Motor Controller
   status = 0;
   HAL_DigitalHandle pwmPort =
-      HAL_InitializePWMPort(HAL_GetPort(2), NULL, &status);
+      HAL_InitializePWMPort(HAL_GetPort(2), nullptr, &status);
```

Comment:
This is a C file. This isn’t valid.

Reply:
Ah true, idk if theres a way to silence clangd about this

Reply:
Depends on what version of C you're using. C23 has nullptr.

Reply:
C23 is barely supported by anything. So definitely not something we can support currently.